gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -629,26 +617,6 @@ def create_network_function_device(self, event):
 
         device_data = self._prepare_device_data_from_nfp_context(nfp_context)
 
-        # dev_sharing_info = (
-        #     orchestration_driver.get_network_function_device_sharing_info(
-        #         device_data))
-
-        # if dev_sharing_info:
-        #     device = self._get_device_to_reuse(device_data, dev_sharing_info)
-        #     if device:
-        #         device = self._update_device_data(device, device_data)
-
-        # # To handle case, when device sharing is supported but device not
-        # # exists to share, so create a new device.
-        # if dev_sharing_info and device:
-        #     # Device is already active, no need to change status
-        #     device['network_function_device_id'] = device['id']
-        #     self._create_event(event_id='DEVICE_HEALTHY',
-        #                        event_data=device,
-        #                        is_internal_event=True)
-        #     LOG.info(_LI("Sharing existing device: %s(device)s for reuse"),
-        #              {'device': device})
-        # REVISIT(TODO): Removing sharing for cisco live demo
```

Comment:
Why we are removing this TODO,
If we need to remove this, then remove "if 0" case also.